package.json

```diff
@@ -6,39 +6,39 @@
   "dependencies": {
     "assemblyscript": "0.19.10",
     "binary-install-raw": "0.0.13",
-    "chalk": "^3.0.0",
-    "chokidar": "^3.0.2",
-    "debug": "^4.1.1",
-    "docker-compose": "^0.23.2",
-    "dockerode": "^2.5.8",
-    "fs-extra": "^9.0.0",
-    "glob": "^7.1.2",
+    "chalk": "3.0.0",
+    "chokidar": "3.0.2",
+    "debug": "4.1.1",
+    "docker-compose": "0.23.2",
+    "dockerode": "2.5.8",
+    "fs-extra": "9.0.0",
+    "glob": "7.1.2",
     "gluegun": "https://github.com/edgeandnode/gluegun#v4.3.1-pin-colors-dep",
```

Comment:
I'll update `gluegun` in another PR, since they've added my fix in the latest version but I need to check if doing a big version jump breaks `graph-cli`.